test/spec/cloudinary-spec.js

```diff
@@ -170,6 +170,10 @@ describe('Cloudinary', function() {
       shorten: true
     }, protocol + '//res.cloudinary.com/test123/iu/test', {});
   });
+  it('Should not set default version v1 to resources stored in folders and ignore the version parameter if exclude_version is set to true', function() {
+    test_cloudinary_url('folder/test', {exclude_version:true}, protocol + '//res.cloudinary.com/test123/image/upload/folder/test', {exclude_version:true});
```

Comment:
@maoznir , isn't that test needs to check with exclude_version=false (or without it altogether)?

Reply:
@maoznir ?